html-forms/html-forms_Finding-a11y-errors/index.html

```diff
@@ -11,22 +11,26 @@
   </head>
   <body>
     <section>
-      <h1 class="title">Understanding A11y in Forms</h1>
+      <h1 id="title">Understanding A11y in Forms</h1>
       <p id="description">
         Please let us know how we can support you with a11y issues.
       </p>
 
-      <form for="title">
-        <fieldset>
+      <form aria-labelledby="title">
+        <fieldset aria-describedby="">
           <legend>Enter your information</legend>
-          <label aria-describedby="name-input">Name:</label>
+          <label aria-describedby="name-input" for="name-Input">Name:</label>
           <input id="name-Input" type="text" name="name" />
-          <label aria-describedby="age-input">Age:</label>
+          <label aria-describedby="age-input" for="age-Input">Age:</label>
           <input id="age-Input" type="text" name="age" />
-          <label aria-describedby="color-input">Color you cannot see:</label>
+          <label aria-describedby="color-input" for="color-Input"
+            >Color you cannot see:</label
+          >
           <input id="color-Input" type="color" name="color" />
-          <textarea rows="7" placeholder="Leave us a message..."></textarea>
-          <button type="submit">Submit</button>
+          <textarea rows="7"></textarea>
```

Comment:
i think here is an `label`-element needed:


< label for="message-input">Leave us a message:< /label>
< textarea id="message-input" rows="7">< /textarea>